core-bundle/contao/templates/twig/backend/data_container/table/search_menu.html.twig

```diff
@@ -0,0 +1,14 @@
+{% trans_default_domain "contao_default" %}
+
+<div class="tl_search tl_subpanel">
+    <strong>{{ 'MSC.search'|trans }}:</strong>
+    <div class="tl_select_wrapper" data-controller="contao--choices">
+        <select name="tl_field" class="tl_select{{ active ? ' active' }}">
```

Comment:
We should use the `HtmlAttributes` class whenever there are dynamic attributes for a HTML tag - not only for readability but security, because then attribute names and values are always properly encoded.
```suggestion
        {% set select_attributes = attrs()
            .set('name', 'tl_field')
            .addClass('tl_select')
            .addClass('active', active)
        %}
        <select{{ select_attributes }}>
```

This also applies for the options below and in the sort menu template.